main.js

```diff
@@ -0,0 +1,152 @@
+// Access posts div
+const postsDiv = document.getElementsByClassName('posts')[0];
+
+// Access message div
+const postMessage = document.getElementById('message');
+
+// Access name div
+const postAuthor = document.getElementById('author');
+
+// Access Submit Post button
+const submitPostButton = document.getElementById('submit-post');
+
+// When the Submit Post button is clicked, create a new div under the posts div.
+// Example: Text - Posted By: Name
+submitPostButton.addEventListener('click', function () {
+  
+  let authorName = postAuthor.value;
+  let messageContent = postMessage.value;
+
+  let postObj = {
+    author: authorName,
+    message: messageContent,
+    comments: [],
+  }
+
+  postAuthor.value = "";
+  postMessage.value = "";
+
+  renderPost(postObj);
+
+});
+
+const renderPost = function(postObj) {
+
+  // Create a new post div render it within postsDiv
+  let newPost = document.createElement('div');
+  newPost.setAttribute('class', 'post');
+
+  // Create a link to delete the post
+  let deletePost = document.createElement('a');

```

Comment:
Suggestion: unless you are reassigning (aka using the assignment operator = ) you can use const when declaring variables